Automattic/wp-calypso
# Gutenboarding: Add design screenshots for the Ibis FSE design

#### Changes proposed in this Pull Request

I've accidentally merged #44972 before updating it with the Ibis FSE design screenshots.
This PR simply adds them.

#### Testing instructions

* Start a new Gutenboarding through `http://calypso.localhost:3000/new?flags=gutenboarding/site-editor`.
* Make sure the Ibis FSE design has a screenshot
